app/javascript/lib/imperial.test.js

```diff
@@ -4624,7 +4631,8 @@ describe('imperial', () => {
           edges: [['a', 'c']],
         });
 
-        const game = new Imperial(board);
+        const logger = new Logger('silent');
+        const game = new Imperial(board, logger);
```

Comment:
Hm should we reinstate the `unittest` logger scope then if it turns out that failing tests become quite noisy

Reply:
Also we'll probably have to update all the `new Imperial`s sadly, probably worth also reinstating the `newImperial` helper again

Reply:
I prefer something like `silent` to clarify that it is not just meant for tests but for particular tests (ie, those that purposefully send invalid actions though). In other cases, the tests were just wrong and I've fixed them. Having those cases fall through to `console.error()` was actually very helpful in resolving the error! But as of this commit, the tests are no longer noisy.

Reply:
>Also we'll probably have to update all the new Imperials sadly, probably worth also reinstating the newImperial helper again

Why? things are passing fine here, no?

Reply:
When the other unittests do fail, it'd be noisy again, no?

Reply:
Fair, just worried about the visibility of the error if and when that happens

For example in this PR's case, technically we didn't fix the issue because the invalid logs are still here, yet we're suppressing the error logs because it isnt really worth the effort, the logs don't contribute much and it's an acceptable "fail"

As long as you're aware of the potential, that's fine. Just food for thought

Reply:
<img width="392" alt="Screenshot 2023-03-28 at 3 32 03 PM" src="https://user-images.githubusercontent.com/10481391/228347282-d1f659a1-e2bd-488d-a49f-9e398b47eae8.png">
This deals specifically with invalid moves. They are necessary to run these tests because it is testing that the system will ignore them. In all other cases, though, we want to see the logs.